### DISCUSSION TITLE
api folder not getting created

### DISCUSSION THREAD

**Discussion by Anil-Kumar-k** [atrocore/atrocore#505] [General]
Hello,
I've installed atro pim without git. installation was successful.when i hit the URL getting 404.When we checked apache2 logs found that it is trying to reach api/v1/Installer/getTranslations. However api folder itself not created. Please help me to resolve this issue. 
Thanks
Anil

**Comment by rratsun** [comment]
Hello,

Maybe this closed issue [#288](https://github.com/atrocore/atrocore/issues/288) will help.

Regards

**Reply by rratsun** [reply]
Please, read [web-server-preparation-apache](https://github.com/atrocore/docs/blob/master/atrocore/admin-guide/web-server-preparation-apache.md) documentation attentively.

**Comment by Anil-Kumar-k** [comment]
Thank you very much for your response and it fixed the issue.

Is there anyway where I can change the DB configuration from existing to new mysql RDS server. 

Thanks in advance for the help.

Thanks & Regards,

Anil Kumar K
Director, Systems Engineering
M: +91 9845538434

Website: www.ergos.in | www.ergoslive.com | www.grainbank.in

Ergos Business Solutions Pvt. Ltd.

**Reply by rratsun** [reply]
1) You should create mysql dump from existing DB first
2) You should import your mysql dump into new mysql server
3) You should change DB configuration in ```/var/www/your_domain/data/config.php```
